tests/program-data/program_data-keypair.json

```diff
@@ -0,0 +1 @@
+[114,99,192,17,48,208,90,184,231,46,220,91,47,115,132,253,218,163,228,101,8,121,220,138,41,140,176,127,254,91,51,28,176,244,174,182,223,57,57,125,117,201,31,213,9,39,207,212,100,173,88,252,61,235,89,156,53,86,4,90,16,251,191,219]
```

Comment:
this exists so it can be copied into the target/deploy folder. see the github actions job `test-program-data`